### PR TITLE
Generic SelectBox component

### PR DESCRIPTION
**Pull request by naomiaz**
Added to:
- PageHeader
- TeamMemberOverview
- ClientsOverview

### CODE REVIEW

File: services/components/select-box/SelectBox.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import PropTypes from 'prop-types';
+
+import './select-box.scss';
+
+const SelectBox = ({
+  className, defaultValue, name, onChangeFunction, options
+}) => (
+  <select
+    className={`select-box ${className}`}
+    name={name}
```
Comment (boris-arkenaar) [inline services/components/select-box/SelectBox.jsx]: This `name` is being used?

File: services/components/select-box/SelectBox.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    onChange={(event) => onChangeFunction(event.target.value)}
+    value={defaultValue}
```
Comment (boris-arkenaar) [inline services/components/select-box/SelectBox.jsx]: This `defaultValue` suggests it's only the initial default value, but it's actually the up-to-date currently selected value, right?

File: services/components/select-box/SelectBox.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+import './select-box.scss';
+
+const SelectBox = ({
+  className, defaultValue, name, onChangeFunction, options
+}) => (
+  <select
+    className={`select-box ${className}`}
+    name={name}
+    onChange={(event) => onChangeFunction(event.target.value)}
```
[the remark this answers is not in the document]
Comment (naomiaz) [reply services/components/select-box/SelectBox.jsx]: Changed this in the next feature branch #46

File: services/components/select-box/SelectBox.jsx (hunk 1 above)
Comment (naomiaz) [reply services/components/select-box/SelectBox.jsx]: Made name not required in next feature branch #46

**Pull request merged**